light-client/src/consensus/consensus_client.rs

```diff
@@ -615,6 +620,32 @@ fn get_bits(bitfield: &BitVector<typenum::U512>) -> u64 {
     count
 }
 
+fn verify_sync_committee_signature(
+    pks: &[PublicKey],
+    attested_header: &BeaconBlockHeader,
+    signature: &BlsSignature,
+    genesis_root: &[u8],
+    fork_version: &[u8],
+) -> bool {
+    let res: Result<bool> = (move || {
+        let pks: Vec<&PublicKey> = pks.iter().collect();
+        let header_root = bytes_to_bytes32(attested_header.tree_hash_root().as_slice());
+        let signing_root = compute_committee_sign_root(genesis_root, header_root, fork_version)?;
+
+        Ok(is_aggregate_valid(
+            signature,
+            signing_root.r#as_bytes(),
+            &pks,
+        ))
+    })();
+
+    if let Ok(is_valid) = res {
+        is_valid
+    } else {
+        false
+    }
+}
```

Comment:
```suggestion
fn verify_sync_committee_signature(
    public_keys: &[PublicKey],
    attested_header: &BeaconBlockHeader,
    signature: &BlsSignature,
    genesis_root: &[u8],
    fork_version: &[u8],
) -> bool {
    let result: Result<bool> = (move || {
        let public_keys: Vec<&PublicKey> = public_keys.iter().collect();
        let header_root = bytes_to_bytes32(attested_header.tree_hash_root().as_slice());
        let signing_root = compute_committee_sign_root(genesis_root, header_root, fork_version)?;

        Ok(is_aggregate_valid(
            signature,
            signing_root.r#as_bytes(),
            &public_keys,
        ))
    })();

    result.unwrap_or_default()
}
```
I think `pks` and `res` shouldn't use shorthand, we aren't saving many characters it just makes the code less readable.

```
        if let Ok(is_valid) = res {
            is_valid
        } else {
            false
        }
```

was updated to `res.unwrap_or_default()` 2 merges ago, it is a bit cleaner so I think we should keep it. It was updated to use `unwrap_or_default()` in this PR https://github.com/ethereum/trin/pull/1447